pysmt/logics.py

```diff
@@ -109,7 +109,7 @@ def combine(self, other):
             real_arithmetic=self.real_arithmetic | other.real_arithmetic,
             integer_difference=integer_difference,
             real_difference=real_difference,
-            linear=self.linear | other.linear,
+            linear=self.linear & other.linear,
```

Comment:
The answer for the first question is the second question ;)
Ok to change them to logical.
On May 16, 2016 11:14 AM, "Andrea Micheli" <notifications@github.com> wrote:

> In pysmt/logics.py
> <https://github.com/pysmt/pysmt/pull/257#discussion_r63369786>:
>
> > @@ -109,7 +109,7 @@ def combine(self, other):
> >              real_arithmetic=self.real_arithmetic | other.real_arithmetic,
> >              integer_difference=integer_difference,
> >              real_difference=real_difference,
> > -            linear=self.linear | other.linear,
> > +            linear=self.linear & other.linear,
>
> Also, why using bitwise operator in the other cases?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pysmt/pysmt/pull/257/files/5bd0403180c346d3183d6dba5d8d3a3cd522bb3a#r63369786>
>